attributes/default.rb

```diff
@@ -11,5 +11,27 @@
 default['macos']['remote_login_enabled'] = true
 default['macos']['disk_sleep_disabled'] = false
 
-default['macos']['network_time_server'] = 'time.windows.com'
+default['macos']['network_time_server'] = 'time.apple.com'
 default['macos']['time_zone'] = 'America/Los_Angeles'
+
+node.default['authorization']['sudo']['groups'] = 'wheel'
```

Comment:
`node` is always implied in this file - is there a specific need for this prefix?